### DISCUSSION TITLE
I'm having problems to save(persist) a file in disc with UploadFile

### DISCUSSION THREAD

**Discussion by gkramer01** [fastapi/fastapi#7512] [Questions]
I need to save a file to disk, but so far I have only been able to clone the file, but without content on it (invalid).
Can anyone help me?

**Comment by phy25** [comment]
Please follow the template to ask the question. It at least asks you to provide the code you are using.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25! :+1: 

And thanks @gkramer01 for closing the issue. If you still have problems, you can create a new one following the template. :nerd_face:
